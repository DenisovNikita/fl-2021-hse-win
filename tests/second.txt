0 to 1 by 1 a
0 to 1 by 1 b
1 to 0 by 1 a
1 to 0 by 1 b
terminal 1
0
