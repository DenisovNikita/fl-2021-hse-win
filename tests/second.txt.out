LexToken(NUM,0,1,0)
LexToken(TO,'to',1,2)
LexToken(NUM,1,1,5)
LexToken(BY,'by',1,7)
LexToken(CHAR,'a',1,10)
LexToken(NUM,0,2,12)
LexToken(TO,'to',2,14)
LexToken(NUM,1,2,17)
LexToken(BY,'by',2,19)
LexToken(CHAR,'b',2,22)
LexToken(NUM,1,3,24)
LexToken(TO,'to',3,26)
LexToken(NUM,0,3,29)
LexToken(BY,'by',3,31)
LexToken(CHAR,'a',3,34)
LexToken(NUM,1,4,36)
LexToken(TO,'to',4,38)
LexToken(NUM,0,4,41)
LexToken(BY,'by',4,43)
LexToken(CHAR,'b',4,46)
LexToken(TERMINAL,'terminal',5,48)
LexToken(NUM,1,5,57)
LexToken(NUM,0,6,59)
